sql/init.sql

```diff
@@ -1,6 +1,6 @@
 
-CREATE TABLE events (
-   id         INTEGER      PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+CREATE TABLE outbox (
```

Comment:
**SIDENOTE:** Not for this PR, but note to me to extend this table with metadata and some default diagnostic tooling like timestamp, trace id, etc.

Reply:
OTEL support would be the next step

Reply:
Added issues:
- https://github.com/oskardudycz/PostgresOutboxPatternWithCDC.NET/issues/8
- https://github.com/oskardudycz/PostgresOutboxPatternWithCDC.NET/issues/9